tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/SqlServerDatabaseMonitoringTests.cs

```diff
@@ -0,0 +1,134 @@
+// <copyright file="SqlServerDatabaseMonitoringTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AdoNet
+{
+    [Trait("RequiresDockerDependency", "true")]
+    [UsesVerify]
+    public class SqlServerDatabaseMonitoringTests : TracingIntegrationTest
```

Comment:
Given this looks largely like a duplicate of the existing SystemDataSqlClient tests, you _might_ want to consider moving these into the existing `SystemDataSqlClientTests` file. The reason being:

- Tests in the same class _won't_ run in parallel, they run sequentially. As it's currently written you _may_ hit concurrency issues (I'm not sure)
- There's a lot of duplicated numbers (e.g. the number of spans generated for various combinations). At some point, these could change, and we'd have to remember to update in both places (which we won't 😅) Moving to the same file and pulling out the commonalities might save a bit of pain later 🙂

Reply:
+1